cellprofiler/modules/calculatestatistics.py

```diff
@@ -4,7 +4,7 @@
 CalculateStatistics
 ===================
 
-**Calculate Statistics** calculates measures of assay quality (V and Z’
+**CalculateStatistics** calculates measures of assay quality (V and Z’
```

Comment:
- Please  remove the not working hyperlink in this top section to making plots

- Please left-align "Here is an example file" at L117 and remove the ticks on the line after it.